static/downloadables/README.md

```diff
@@ -85,7 +85,7 @@ Traits and studies available through this tool can be searched from the PRSKB CL
 ### Additional Optional Parameters
 
 * **-v verbose result file** -- Adding the **-v** parameter will return the output file in a 'verbose' format, which includes a line for each sample/study/trait combination. Additional columns are added that display lists of protective variants, risk variants, variants that are present but do not include the risk allele, and variants that are in high linkage disequilibrium whose odds ratios are not included in the calculations. *NOTE: This only applies to TSV output files. JSON output files are always 'verbose'.*
-* **-g defaultSex** -- This parameter will set the default sex for the samples in the input file. Though a rare occurence, some studies have duplicates of the same SNP that differ by which biological sex the p-value is associated with. You can indicate which sex you would like SNPs to select when both options (M/F) are present. The system default is Female."
+* **-g sex dependent associations** -- This parameter will indicate the sex of the samples in the input file. Though a rare occurence, some studies have duplicates of the same SNP that differ by which biological sex the p-value is associated with. You can indicate which sex you would like SNPs to select when both options (M/F) are present. The system default is to exclude the snp from calculations."
```

Comment:
To make this more clear, change the last sentence to "The system default is to exclude sex-dependent SNPs from calculations."